src/components/EmojiPicker/EmojiPickerMenu/index.native.js

```diff
@@ -126,31 +78,32 @@ function EmojiPickerMenu({preferredLocale, onEmojiSelected, preferredSkinTone, t
      * @param {Object} item
      * @returns {*}
      */
-    const renderItem = ({item}) => {
-        const {code, types} = item;
-        if (item.spacer) {
-            return null;
-        }
+    const renderItem = useCallback(
+        ({item, target}) => {
+            const {code, types} = item;
+            if (item.spacer) {
+                return null;
+            }
+
+            if (item.header) {
+                return (
+                    <View style={[styles.emojiHeaderContainer, target === 'StickyHeader' ? styles.mh4 : {width: windowWidth}]}>
+                        <Text style={styles.textLabelSupporting}>{translate(`emojiPicker.headers.${code}`)}</Text>
+                    </View>
+                );
+            }
+
+            const emojiCode = types && types[preferredSkinTone] ? types[preferredSkinTone] : code;
 
-        if (item.header) {
             return (
-                <View style={styles.emojiHeaderContainer}>
-                    <Text style={styles.textLabelSupporting}>{translate(`emojiPicker.headers.${code}`)}</Text>
-                </View>
+                <EmojiPickerMenuItem
+                    onPress={singleExecution((emoji) => onEmojiSelected(emoji, item))}
+                    emoji={emojiCode}
+                />
             );
-        }
-
-        const emojiCode = types && types[preferredSkinTone] ? types[preferredSkinTone] : code;
-
-        return (
-            <EmojiPickerMenuItem
-                onPress={singleExecution((emoji) => onEmojiSelected(emoji, item))}
-                emoji={emojiCode}
-            />
-        );
-    };
-
-    const isFiltered = allEmojis.length !== filteredEmojis.length;
+        },
+        [styles, windowWidth, preferredSkinTone, singleExecution, onEmojiSelected, translate],
+    );
 
     return (
         <View style={styles.emojiPickerContainer}>
```

Comment:
Can you check what happens if we also pass `StyleUtils.getEmojiPickerStyle(isSmallScreenWidth)` to this view style on native? It seems like it should be safe to pass on both web and native since its return value is determined by `isSmallScreenWidth`

Reply:
Checked it and looks like it's safe to add it here ✅